examples/currency-transfer.yaml

```diff
@@ -0,0 +1,177 @@
+    share:
+      - owner: https://alice.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 150
+      - owner: https://zoe.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 50
+
+  - '@id': urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    '@type': EconomicResource
+    skos:note: credits in transit
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+    share:
+      - owner: https://alice.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 100
+
+  # actual ownership transfer happens only in this flow
+  - '@id': notary:da8f3360-3a6d-4ea8-b57f-4d4236fe2c3e
+    '@type': EconomicEvent
+    action: own
+    provider: https://alice.example/
+    recipient: https://bob.example/
+    resourceInventoriedAs: urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    actsOnQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    '@type': EconomicResource
+    skos:note: credits in transit
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+    share:
+      - owner: https://bob.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 100
+
+  - '@id': bbank:57f01dbe-fd78-40d0-9278-60005debfba6
+    '@type': Process
+    skos:note: credit Bob's account
+
+  - '@id': bbank:33f11508-92b3-4095-9e03-6678cddf7adf
+    '@type': EconomicEvent
+    action: split
+    inputOf: bbank:57f01dbe-fd78-40d0-9278-60005debfba6
+    resourceInventoriedAs: urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    provider: https://bob.example/
+    recipient: https://bob.example/
+    actsOnQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': bbank:33f11508-92b3-4095-9e03-6678cddf7adf
+    '@type': EconomicEvent
+    action: merge
+    outputOf: bbank:57f01dbe-fd78-40d0-9278-60005debfba6
+    resourceInventoriedAs: bbank:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    provider: https://bob.example/
+    recipient: https://bob.example/
+    actsOnQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': bbank:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    '@type': EconomicResource
+    skos:note: Bob's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 600
+
+  - '@id': urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    '@type': EconomicResource
+    skos:note: credits in transit
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:numericValue: 0
+    share:
+      - owner: https://bob.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 300
+      - owner: https://sam.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 300
```

Comment:
Should those shares be zero?

Reply:
oops, pasted on wrong resource... fixed!